src/sofa/pbrpc/rpc_server.cc

```diff
@@ -79,6 +79,16 @@ bool RpcServer::IsListening()
     return _impl->IsListening();
 }
 
+bool RpcServer::RegisterWebServlet(const std::string& path, Servlet servlet)
+{
+    return _impl->RegisterWebServlet(path, servlet);
+}
+
+bool RpcServer::UnregisterWebServlet(const std::string& path)
```

Comment:
it is better for UnregisterWebServlet() to return Servlet, and returns null if not found.